tel-search-connector-demo/src_hd/com/axonivy/connector/telsearch/demo/AddressFinder/AddressFinder.xhtml

```diff
@@ -0,0 +1,51 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/frame-9.xhtml">
+		<ui:define name="title">AddressFinder</ui:define>
+		<ui:define name="content">
+
+			<h3>
+				<a href="https://tel.search.ch">tel.search.ch</a> based address
+				finder
+			</h3>
+
+			<h:form id="form">
+				<p:messages />
+				<h:panelGrid columns="3">
+					<p:outputLabel value="Search" for="Search" style="float:left" />
+					<p:inputText value="#{data.search}" id="Search" required="false"
+						style="float:left" />
+
+					<p:commandButton id="Find" actionListener="#{logic.find}"
+						ajax="false" value="Find" update="form" icon="ui-icon-search" />
+					<p:spacer />
+
+				</h:panelGrid>
+				<br />
+				<p:orderList value="#{data.matches}" var="match" responsive="true"
```

Comment:
On the UI I'd streamline the wording with the official tel.search.ch ... therefore I'd rather speak of 'Entries' than 'Matches
![Selection_1009](https://user-images.githubusercontent.com/15085693/141749160-aace2a10-5c59-457a-adf2-e57b9ddb2835.png)
'